### PR TITLE
add cn doc of functional ctc_loss and CTCLoss class.

### PR DESCRIPTION
**Pull request by lfchener**
add cn doc of functional ctc_loss and CTCLoss class.

### CODE REVIEW

**Comment by lfchener** [comment]
image
image
image

**Comment by lfchener** [comment]
image
image
image

File: doc/fluid/api_cn/nn_cn/functional_cn/ctc_loss_cn.rst [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+                        [1, 2, 2]]).astype("int32")
+        input_lengths = np.array([5, 5]).astype("int64")
+        label_lengths = np.array([3, 3]).astype("int64")
+
+        paddle.disable_static()
+        log_probs = paddle.to_variable(log_probs)
+        labels = paddle.to_variable(labels)
+        input_lengths = paddle.to_variable(input_lengths)
+        label_lengths = paddle.to_variable(label_lengths)
+
```
Comment (swtkiwi) [inline doc/fluid/api_cn/nn_cn/functional_cn/ctc_loss_cn.rst:67]: paddle.to_variable->paddle.to_tensor

Comment (swtkiwi) [reply doc/fluid/api_cn/nn_cn/functional_cn/ctc_loss_cn.rst:67]: 63行-66行都需要改哈

File: doc/fluid/api_cn/nn_cn/loss_cn/CTCLoss_cn.rst [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                        [1, 2, 2]]).astype("int32")
+        input_lengths = np.array([5, 5]).astype("int64")
+        label_lengths = np.array([3, 3]).astype("int64")
+
+        paddle.disable_static()
+        log_probs = paddle.to_variable(log_probs)
+        labels = paddle.to_variable(labels)
+        input_lengths = paddle.to_variable(input_lengths)
+        label_lengths = paddle.to_variable(label_lengths)
+
```
Comment (swtkiwi) [inline doc/fluid/api_cn/nn_cn/loss_cn/CTCLoss_cn.rst:68]: 64-67行：
paddle.to_variable->paddle.to_tensor

Comment (lfchener) [reply doc/fluid/api_cn/nn_cn/loss_cn/CTCLoss_cn.rst:68]: done!

File: doc/fluid/api_cn/nn_cn/functional_cn/ctc_loss_cn.rst (hunk 1 above)
Comment (lfchener) [reply doc/fluid/api_cn/nn_cn/functional_cn/ctc_loss_cn.rst:67]: done!

**Review by swtkiwi — APPROVED** [review]
LGTM

**Pull request merged**